README.md

```
@@ -93,6 +93,14 @@ Requirements:
 * On OS X, verify that `homebrew` is installed and run: `brew install cmake m4 automake peg libtool autoconf`.
     * The version of Clang that ships with the OS X toolchain does not support OpenMP, which is a requirement for FalkorDB. One way to resolve this is to run `brew install gcc g++` and follow the on-screen instructions to update the symbolic links. Note that this is a system-wide change - setting the environment variables for `CC` and `CXX` will work if that is not an option.
 
+* Make sure your `automake` and `aclocal` versions are >= `1.14`
+    * Create symbolic links for both `automake` and `aclocal`
```

Markdownlint (0.37.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)